scripts/sell_unused_slots.py

```diff
@@ -41,6 +45,42 @@
 log = logging.getLogger(__name__)
 
 
+@attr.s(auto_attribs=True, kw_only=True, frozen=True)
+class Lambda:
+    name: str
+    slot_location: str
+
+    @property
+    def is_contribution_lambda(self) -> bool:
+        for lambda_name in self._contribution_lambda_names():
+            try:
+                # FIXME: Eliminate hardcoded separator
+                #        https://github.com/databiosphere/azul/issues/2964
+                resource_name, _ = config.unqualified_resource_name(self.name,
+                                                                    suffix='-' + lambda_name)
+            except RequirementError:
+                pass
+            else:
+                if resource_name == 'indexer':
+                    return True
+        return False
+
+    @classmethod
+    @cache
+    def _contribution_lambda_names(cls) -> FrozenSet[str]:
+        indexer = load_app_module('indexer')
+        return frozenset((
```

Comment:
I already know a PR were this would break: #3622

```python
from azul.modules import load_app_module
app = load_app_module('indexer')
app.contribute.queue
'azul-notifications-prod'
from azul import config
config.notifications_queue_name()
'azul-notifications-prod'
```

So look for handlers whose unqualified value of the `queue` attribute is the same as the unqualified name of the notifications queue.